packages/bootstrap-4/test/helpers/createMocks.ts

```diff
@@ -0,0 +1,33 @@
+import { WidgetProps } from "@rjsf/core";
+import { JSONSchema7 } from "json-schema";
+
+export const mockSchema: JSONSchema7 = {
+  type: "array",
+  items: {
+    type: "string",
+  },
+};
+
+export const mockEventHandlers = (): void => void 0;
+
+export function makeWidgetMockProps(props: Partial<WidgetProps>): WidgetProps {
```

Comment:
The `props` argument should default to `{}` so we don't need to pass it an empty object if no override props are needed.